### PR TITLE
Multi forecast 2

### PR DESCRIPTION
**Pull request by aaron-kaplan**
Supports multiple instances of the flex forecast maproom.

I have to do this for Lesotho urgently. I don't recommend adapting any other countries yet, as I plan to do more simplification once the Lesotho deadline is taken care of.

Probably easiest to read one commit at a time, and suppressing whitespace differences when viewing the commit "Support multiple instances of flex forecast maproom."

AccompanyingLesotho PR: https://bitbucket.org/iridl/python_maproom_lesotho/pull-requests/3

### CODE REVIEW

**Review by remicousin — COMMENTED** [review]
I was able to build the multi-page with the 2 forecasts and each page seemed fine except I am unable to read any data. The code fails at the concatenation step where it finds nothing to concatenate. I suspected maybe a permission issue with your files so I tried to read a copy I had, except I had only one set, and I failed to fall back to reading only one set for one page. I tried to give the same path to both pages but that failed too. Then I copied your data to my space and gave large permissions but that still fell back to the concatenation issue.

I am out of ideas to figure out what's going on.

**Comment by aaron-kaplan** [comment]
Sorry about that, fixed now. Needed to create the subdirectories pnep-djf-v2/djf and pnep-ond-v2/ond and move everything else into those. I was working on my laptop with a local copy of the data, and forgot that I had made that change locally.

I'm running with
```
CONFIG=config.yaml:../../python_maproom_lesotho/config.yaml:../../python_maproom_lesotho/config-iri.yaml:config-local.yaml
```
where `config-local.yaml` defines `db.password` and `dev_server_port`.

File: enacts/flex_fcst/maproom.py [hunk 1]
```diff
@@ -20,700 +20,775 @@
...
+            Output("lat_input_tooltip", "children"),
+            Output("lng_input", "min"),
+            Output("lng_input", "max"),
+            Output("lng_input_tooltip", "children"),
+            Output("map", "center"),
+            State("lead_time_label", "style"),
+            State("lead_time_control", "style"),
+            Input("location", "pathname"),
+    )
+    def initialize(lead_time_label_style, lead_time_control_style, path):
```
[the remark this answers is not in the document]
Comment (aaron-kaplan) [reply enacts/flex_fcst/maproom.py:168]: The value isn't used, but it causes this callback to be triggered on initial page load.

If/when we start putting values of the controls (menu selections etc.) in the URL, we will get those from  `path`.

[the remark this answers is not in the document]
Comment (aaron-kaplan) [reply enacts/flex_fcst/maproom.py:168]: (Having this callback trigger on initial page load is what allowed me to stop loading data at import time, which no longer makes sense now that we're using the same code with multiple datasets.)

**Review by remicousin — APPROVED** [review]
Works all fine now.

**Comment by aaron-kaplan** [comment]
@remicousin calling your attention to my additions in the CHANGELOG in the last commit.

**Pull request merged**